CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+
+## 2.8.1
+Change the response format for some clients (emails, analytics and landing_pages) to include headers.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** I noticed  that we've used the _past form_ to list the changes, so I think we can rewrite this sentence to:

‏
```suggestion
Changed the response format for some clients (emails, analytics, and landing pages) to include headers.
```